Ladybird/BrowserWindow.cpp

```diff
@@ -110,6 +110,11 @@ BrowserWindow::BrowserWindow(Browser::CookieJar& cookie_jar, StringView webdrive
     view_menu->addAction(open_previous_tab_action);
     QObject::connect(open_previous_tab_action, &QAction::triggered, this, &BrowserWindow::open_previous_tab);
 
+    auto* open_history_window_action = new QAction("Open &History Window", this);
+    open_history_window_action->setShortcut(QKeySequence(Qt::CTRL | Qt::Key_H));
```

Comment:
This keyboard shortcut doesn't currently work because it conflicts with the one for "Dump History", defined on line 245. It makes sense to use Ctrl-H here, so change that other one, and we should be good to go. :+1: